test/calibration/experiments/test_rabi.py

```diff
@@ -198,7 +198,9 @@ def simulate_experiment_data(self, thetas, amplitudes, shots=1024):
             circuits.append(qc)
 
         sim = QasmSimulatorPy()
-        result = execute(circuits, sim, shots=shots, seed_simulator=10).result()
+        circuits = transpile(circuits, basis_gates=["rz", "sx", "x", "cx"])
```

Comment:
Why do you need the basis gates, considering that they were not present before?

Reply:
You're right, I don't need to - only to pass the backend. Fixed.